tests/test_connection.c

```diff
@@ -1036,152 +1041,6 @@ void test_create_nopoll_connection()
 }
 
 
-void test_get_interface_down_event()
```

Comment:
Why is this test deleted?

Reply:
I thought we weren't handling interface down in this update.

Reply:
These are existing interface down tests already added